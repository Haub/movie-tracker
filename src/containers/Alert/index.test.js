import React from 'react';
import { shallow } from 'enzyme';

import { Alert } from '.';

describe('Alert', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Alert />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should display an alert message', () => {
    const wrapper = shallow(<Alert alertMsg={'Alert!'} />);
    expect(wrapper.find('h4').text()).toEqual('Alert!');
  });
});
