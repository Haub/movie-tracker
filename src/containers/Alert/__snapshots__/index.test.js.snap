// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should match snapshot 1`] = `<h4 />`;
